docs/open-source/installation.mdx

```diff
@@ -0,0 +1,121 @@
+---
+id: installation
+hide_title: true
+sidebar_label: Installation
+slug: /open-source/installation
+---
+
+# Open source Installation details
```

Comment:
Let's make sure it's clear that this guide is for the Docs Website Installation.